### PR TITLE
[2/2] Build virtualenv into the datadog-agent package

### PR DESCRIPTION
**Pull request by adamstegman**
Under certain circumstances (using [bosh-init](http://bosh.io/docs/using-bosh-init.html)) the virtualenv was not set up correctly and was loading an empty Python environment. So no dependencies were loaded and the agent would not start.

I'm getting rid of the separate virtualenv package and just setting it up within datadog-agent. These packages didn't really need to be separate; they're not reused in any way, and setting up virtualenv just once makes it simpler to get it in the right place.

### CODE REVIEW

File: packages/datadog-agent/packaging [hunk 1]
```diff
@@ -11,9 +11,10 @@ AGENT_VERSION="5.8.5"
 DD_START_AGENT=0
 PIP_VERSION=6.1.1
 SETUPTOOLS_VERSION=20.9.0
+VIRTUALENV_VERSION="1.11.6"
 
-mkdir -p $BOSH_INSTALL_TARGET/bin
-cp -LR $DEPENDENCY_DIR/virtualenv $BOSH_INSTALL_TARGET/venv
+mkdir -p $BOSH_INSTALL_TARGET/venv
+python datadog-agent/virtualenv-$VIRTUALENV_VERSION.py --no-pip --no-setuptools $BOSH_INSTALL_TARGET/venv
```
Comment (jeremiemv) [inline packages/datadog-agent/packaging]: python all the things?

Comment (adamstegman) [inline packages/datadog-agent/packaging]: Inlined code from the deleted package

Comment (jeremiemv) [inline packages/datadog-agent/packaging]: Yep, read the PR description ;)

**Comment by vcardillo** [comment]
[2/2] 🚢

**Pull request merged**